examples/src/fibonacci/fib2/prover.rs

```diff
@@ -3,20 +3,28 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+use std::marker::PhantomData;
+
+use winterfell::crypto::ElementHasher;
```

Comment:
nit: `ElementHasher` should come from `super` (and merged with the line below); `PhantomData` should come from core.